lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenCompletionParticipant.java

```diff
@@ -265,13 +265,6 @@ public void onXMLContent(ICompletionRequest request, ICompletionResponse respons
 		case "goal":
 			collectGoals(request).forEach(response::addCompletionItem);
 			break;
-		case "configuration":
```

Comment:
@mickaelistria it seems this wasn't necessary, tests are passing without it and things seemed to work when user testing. Any worries about removing this?